x-pack/plugins/lens/public/visualizations/datatable/components/columns.tsx

```diff
@@ -260,10 +262,13 @@ export const createGridColumns = (
       }
     }
     const currentAlignment = alignments && alignments[field];
-    const hasMultipleRows = headerRowHeight === 'auto' || headerRowHeight === 'custom';
+    const hasMultipleRows =
+      headerRowHeight === RowHeight.Auto ||
+      headerRowHeight === RowHeight.Custom ||
+      headerRowHeight === undefined;
```

Comment:
nit: just a bit shorter and easy to extend

```suggestion
    const hasMultipleRows =[RowHeight.Auto, RowHeight.Custom, undefined].includes(headerRowHeight);
```